driver/bpf/types.h

```diff
@@ -206,6 +206,8 @@ struct sysdig_bpf_settings {
 	bool dropping_mode;
 	bool is_dropping;
 	bool tracers_enabled;
+	uint16_t fullcapture_port_range_start;
+	uint16_t fullcapture_port_range_end;
```

Comment:
It seems like these values are never initialized in scap. BPF maps are not automatically cleared, so we essentially get garbage by default. Please look at the `set_default_settings` function in `scap_bpf.c`.